source/extensions/common/wasm/v8/v8.cc

```diff
@@ -153,6 +164,17 @@ template <typename T> struct V8ProxyForGlobal : Global<T> {
   wasm::Global* global_;
```

Comment:
Now, that you reminded me about `ConvertWordTypeToUint32`, you can change this implementation to:
```
T get() override { return global_->get().get<typename ConvertWordTypeToUint32<T>::type>(); };
void set(const T& value) override { global_->set(makeVal(value)); }
```
and get rid of the specialized `V8ProxyForGlobal<Word>`.

But you need to move it around, so that it's defined after `makeVal`.

Reply:
done.